README.md

```diff
@@ -49,9 +49,9 @@ Come find us on the [development community chat](https://zulip.com/development-c
 - **Checking Zulip out**. The best way to see Zulip in action is to drop by the
   [Zulip community server](https://zulip.com/development-community/). We also
   recommend reading about Zulip's [unique
-  approach](https://zulip.com/why-zulip/) to organizing conversations.
+  approach](https://zulip.com/why-zulip/) to organize conversations.
```

Comment:
I think "organizing" is actually correct here. We may want to do a further tweak to make the language better.